themes/digital.gov/layouts/events/single.html

```diff
@@ -49,17 +49,15 @@
 
       {{/* EVENT Actions — REGISTER and Add to Calendar */}}
       {{- if eq $future_event true -}}
-        <div class="grid-container-desktop">
-          <div class="actions">
-            {{/* Register */}}
-            <a
-              aria-label="Register for {{- .Params.title -}}"
-              class="btn btn-register"
-              href="{{- .Params.registration_url -}}"
-              onclick="__gaTracker('send', 'event', 'outbound-article', '{{- .Params.registration_url -}}', 'REGISTER NOW');"
-              >Register</a
-            >
-          </div>
+        <div class="actions">
```

Comment:
**Question/Suggestion**
So i see this class and made me wonder what its doing and its getting styled here `themes/digital.gov/src/scss/new/_buttons.scss` any thought to us needed to update this class name to be more BEMing at all?

Reply:
Yup, that needs to be fixed

Reply:
Fixed with d040908c65a847ea1470849b029a17ef24c8ce04